site/content/posts/contributor-of-the-month.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introducing the Contributor of the Month initiative"
+slug: contributor-of-the-month
+date: 2022-03-01
+author: Cartographer Team
+authorLocation: https://github.com/vmware-tanzu/cartographer/blob/main/MAINTAINERS.md
+image: /img/posts/contributor-of-the-month/Contributor-of-the-month-v3.png
+excerpt: "A monthly program to reward contributions"
+tags: ['community']
+---
+
+Since October 2021, when the Cartographer project was made public, we have been very fortunate to count with individuals who have helped improve the codebase and docs while also telling the Cartographer story in conferences, live streams and other forms of content. All of them have done this out of intrinsic motivation, without expecting any reward or public recognition and we are forever grateful to them for this.
+
+Nevertheless, we would like to fill in the [extrinsic motivation](https://www.rochester.edu/emerging-leaders/understanding-intrinsic-and-extrinsic-motivation/#:~:text=Intrinsic%20motivation%20involves%20performing%20a,punishment%20or%20receiving%20a%20reward) void by rolling out a small initiative called **Contributor of the Month**
+
+**How does it work?**
+
+We plan to recognize contributors in three different categories:
+
+* Code contributor (merged PRs)
+* Content creator (tweets/blog posts/screencasts/presentations/podcast)
+* Project and community supporter (community meetings participation/Slack interactions)
+
+Code contributions will be tracked using the `contributor shoutouter` program located in [this repo](https://github.com/vmware-tanzu/community-engagement/tree/main/shoutouter) while content and community support will be tracked down by the Cartographer community team.
```

Comment:
Have we got a list of who is in the "Cartographer community team?" (I'm guessing it's you David :) ) - Is it something we can link to? (perhaps something we should do as we pull the Docs site out to cartographer-site? Make sure when we refer to teams and individuals, there's a directory that folks can refer to, know who to reach out to?

Reply:
You're right, maybe a TEAMS file like Paketo has: https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md